sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/BuilderHelper.java

```diff
@@ -176,8 +168,8 @@ private static String getBearerTokenScope(UrlBuilder endpoint) {
      */
     private static HttpPipelinePolicy getResponseValidationPolicy() {
         return new ResponseValidationPolicyBuilder()
-            .addOptionalEcho(Constants.HeaderConstants.CLIENT_REQUEST_ID)
-            .addOptionalEcho(Constants.HeaderConstants.ENCRYPTION_KEY_SHA256)
+            .addOptionalEcho(StorageConstants.HeaderConstants.CLIENT_REQUEST_ID)
+            .addOptionalEcho(StorageConstants.HeaderConstants.ENCRYPTION_KEY_SHA256)
```

Comment:
Is it required to validate that the response headers are the same as the request headers?

Reply:
I'm not sure, this was added before my time in Tables. I suspect it's copying what's done in the Storage libraries where this is validated. I can check with the rest of the team to see if other languages are doing the same validation.

Reply:
It seems like what I said above was the case. The other languages are not doing this at all, so I think we can remove it for the time being.